trio/_dtls.py

```diff
@@ -841,10 +841,7 @@ def __init__(self, endpoint: DTLSEndpoint, peer_address: Any, ctx: Context):
         # support and isn't useful anyway -- especially for DTLS where it's equivalent
         # to just performing a new handshake.
         ctx.set_options(
-            (
-                SSL.OP_NO_QUERY_MTU
-                | SSL.OP_NO_RENEGOTIATION  # type: ignore[attr-defined]
-            )
+            SSL.OP_NO_QUERY_MTU | SSL.OP_NO_RENEGOTIATION  # type: ignore[attr-defined]
```

Comment:
is this formatting change due to a version bump?

Reply:
It's changed by black funnily enough, not ruff.
Tested with reverting changes and running pre-commit locally.

Reply:
Well, it's a good change - but I'm confused why it's in this PR and not in a separate PR that does the actual version bump in the committed config file.